superset-frontend/src/explore/components/controls/AnnotationLayer.jsx

```diff
@@ -455,7 +455,6 @@ export default class AnnotationLayer extends React.PureComponent {
         <div style={{ marginRight: '2rem' }}>
           <PopoverSection
             isSelected
-            onSelect={() => {}}
             title="Annotation Slice Configuration"
             info={`This section allows you to configure how to use the slice
                to generate annotations.`}
```

Comment:
```suggestion
            info={t(`This section allows you to configure how to use the slice
               to generate annotations.`)}
```
I know this is just bycatch, but if you get in there for my other pedantic nits, we might as well get these too ;)